tests/sentry/models/test_organization.py

```diff
@@ -58,6 +82,30 @@ def test_slugify_long_org_names(self):
         assert len(org2.slug) > len("stove-electrical-and-cateri-")
         assert org.slug != org2.slug
 
+    def test_slugify_long_org_names_within_region(self):
+        if SiloMode.get_current_mode() != SiloMode.REGION:
+            return
+
+        org = Organization.objects.create(name="Stove, Electrical, and Catering Stuff")
+        assert org.slug == "r-na-stove-electrical-and-cate"
+
+        # Ensure org slugs are unique
+        org2 = Organization.objects.create(name="Stove, Electrical, and Catering Stuff")
+        assert org2.slug.startswith("r-na-stove-electrical-and-")
+        assert len(org2.slug) > len("r-na-stove-electrical-and-")
+        assert org.slug != org2.slug
```

Comment:
Do we have a test covering two slugs that would be the same up until the max length?

Reply:
I believe that's what this test is doing unless I'm misunderstanding the case you're describing.

Reply:
My bad, I misread the test.